src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier extends Figure {
+
+    public static final int FIGURE_COUNT = 5;
+    private Random random = new Random();
+
+    @Override
+    public double getSide() {
+        double sideSize = random.nextInt(FIGURE_COUNT) + 1;
+        return sideSize;
+    }
+
+    public int getRandomFigure() {
```

Comment:
```suggestion
    public Figure getRandomFigure() {
```
цей метод повинен повертати рандомно згенеровану фігуру